SU-SWS/stanford_media
# D8CORE-2428 Patch drupal core for media library validation

# READY FOR REVIEW

# Summary
- Patch drupal core for media library validation
- see https://www.drupal.org/project/drupal/issues/3059955
- Stanford_profile has an acceptance test for it too https://github.com/SU-SWS/stanford_profile/pull/328

# Need Review By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. use the patch from https://www.drupal.org/project/drupal/issues/3059955#comment-13405553
1. go to create a page with a media field
1. select an image but don't click to insert it
1. upload or search for another image
1. choose a second image
1. click to insert it
1. verify you are forced to only choose the correct number of items.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
